### ISSUE TITLE
Mining cyborg weird interactions with hyper chassis KA

### ISSUE THREAD

**Issue by TheWiznard** [tgstation/tgstation#31894]
Round ID: 77615

So it's a long lowpop round and I eventually bring back some minerals for upgrades at robotics, I get lavatracks, selfheal, vtec and cooldown increase. This lasts me a trip for maybe 20-30 minutes when I come back and get a satchel of holding, when the roboticist decides since we can't get the KA aoe upgrade that I wanted because we didn't have it researched yet we'd just go buy it from the vendor. At this point we get to cargo and buy the aoe upgrade and install it, and then decide to try out the other upgrades like the hyper chassis. 

When I get back to mining however I find that the KA behaves as if it isn't upgraded at all, but it still shows the effects of the upgrade happening. Such as the aoe blast appearing but only breaking one tile. I'm not sure if this is because the chassis was installed after the other upgrades or not though because when I attempted to have the roboticist help me out again he put them back in the same order and ran off. 

tl;dr : installing a chassis upgrade to the cyborg KA works but appears to break all previous installed upgrades.

**Comment by ChangelingRain** [comment]
The AoE upgrade you can buy is mob only.

**Comment by TheWiznard** [comment]
Ah okay so borg upgrade is mining, gotcha thanks didn't know the difference
